enterprise/cmd/frontend/internal/auth/init.go

```diff
@@ -47,6 +49,7 @@ func Init(logger log.Logger, db database.DB) {
 		githuboauth.Middleware(db),
 		gitlaboauth.Middleware(db),
 		bitbucketcloudoauth.Middleware(db),
+		azureoauth.Middleware(db),
```

Comment:
Can we repeat the alphabetical ordering here? I think it might make sense for readability purposes:
```suggestion
		azureoauth.Middleware(db),
		bitbucketcloudoauth.Middleware(db),
		githuboauth.Middleware(db),
		gitlaboauth.Middleware(db),
```